src/Hit/Issue.hs

```diff
@@ -164,3 +162,24 @@ parseOwnerRepo url =
 
     stripGitSuffix :: Text -> Maybe Text
     stripGitSuffix x = whenNothing (T.stripSuffix ".git" x) (Just x)
+
+data AssignSuccess = AlreadyAssigned | NewlyAssigned
+
+assignToIssue :: Issue -> IO ()
+assignToIssue issue = withOwnerRepo assignAction >>= \case
+    Left err              -> errorMessage (T.pack $ show err) >> exitFailure
+    Right AlreadyAssigned -> successMessage "Already assigned to issue"
```

Comment:
Maybe this should be in another colour than `successMessage`?

Reply:
Yes. I think `infoMessage` makes sense here!